bench/string-endsWith.js

```diff
@@ -0,0 +1,42 @@
+const { createBenchmarkSuite } = require('../common')
+
+const suite = createBenchmarkSuite('Check if string ends with a substring')
```

Comment:
```suggestion
const suite = createBenchmarkSuite('endsWith comparisson')
```

Let's name it as operation